go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/elasticsearch-operator
 
-go 1.14
+go 1.15
```

Comment:
is this required as part of the CVE fix?

Reply:
Nope this one slipped in.